volumes/rescu/rescu_sensors_eval_readme.md

```diff
@@ -0,0 +1,59 @@
+This PR accomplishes identification of network wide and individual detector outages.
+
+1) **volumes/rescu/create-view-network-outages.sql** - Creates a table of network wide RESCU outages (no values from any sensor over any duration). 
+This can be used to find good dates for a data request or to be part of an alert pipeline. 
+For example, here is some sample code to start with a list of eligible dates for a request and exclude any date with a network outage of any length:
+
+```
+--use case: find dates and times when there were no network outages: 
+--there are 34 dates so far in 2023 with no network wide outages. 
+WITH list_dates AS (
+    SELECT generate_series('2023-01-01', '2023-06-12', '1 day'::interval)::timestamp AS date
+)
+
+SELECT l.date 
+FROM list_dates AS l
+LEFT JOIN gwolofs.network_outages AS nout ON
+    nout.date_start <= l.date AND
+    nout.date_end >= l.date
+WHERE nout.date_start IS NULL
+```
+
+2) **volumes/rescu/create-mat-view-individual-outages.sql** - identify individual detector outages. Could be useful for future data requests.
+Similar to network outages but for each individual detector. Currently network wide and individual outages overlap due to difficulty of separating overlapping datetime ranges. 
+
+Here is a sample query which finds a list of dates where a list of sensors are all active with no individual outages. 
```

Comment:
This line appears tabbed back to the left, if you want it to appear in line with the text in the numbered list, it needs to be 3 spaces over.

1. Testing

   testing a second paragraph

2. Testing a new item